New post: Server softwares (#43)
FalixNodes-Software/help-center: _posts/minecraft/general/2021-8-22-server-softwares.md

---
layout: post

> title:  "Server softwares"

Reviewer: Capitalize the s in "softwares"

categories: Minecraft
tag: mcgeneral

> permalink: /minecraft/general/server-software/

Reviewer: This is an article that talks about multiple server softwares. I think that the permalink should be `/minecraft/general/server-softwares/`

Author: We should leave it Minecraft, we can move it once other server softwares are added

Reviewer: I didn't move it from Minecraft, I simply added an s to make it plural.

Author: Oh my bad

---

> # Server softwares

Reviewer: Capitalize the s in "softwares"

> 

Reviewer: I believe that adding a quick description to this article is important. I came up with:

Choosing your server's software is a very important task. This article will help you choose the software that fits your server's needs.

> ## 1. PaperMC

Reviewer: There's no need for an ordered list in this article. I think that removing the numeration will help make it look cleaner as well.

Author: The numbers in the list correspond to the number in the server setup script

Reviewer: Ah, understood.

> Paper MC is a fork of bukkit and spigot that is optimized and allows you to use plugins, both spigot and bukkit plugins work, It also fixes some Vanilla bugs and exploits!

Reviewer 2: ```suggestion
[PaperMC](https://papermc.io) is a fork of Spigot that is optimized and allows you to use Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!
```

## 2. Purpur

> Purpur is a Fork of Paper and Tuinity that is very optimized, it has many extra features and supports spigot and bukkit plugins. We recommend using this server software.

Reviewer 2: ```suggestion
[Purpur](https://purpur.pl3x.net/) is a Fork of Tuinity that is very optimized. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.
```

## 3. Fabric

> Fabric is a Minecraft software that supports fabric mods and is lightweight.

Reviewer 2: ```suggestion
[Fabric](https://fabricmc.net/) is a lightweight Minecraft software used for Fabric mods.
```

## 4. Vanilla

> Vanilla is the default Minecraft software, it does not support plugins or mods.

Reviewer 2: ```suggestion
Vanilla is the default software provided by Minecraft. It does not support plugins or mods.
```

> ## 5. Vanilla snapshot
> Vanilla snapshot is the default Minecraft server, it does not support plugins or mods, use this if you want your server's version to be a snapshot.

Reviewer 2: ```suggestion
## 5. Vanilla Snapshot
Vanilla Snapshot is a testing version of Minecraft. Snapshot's usually contain bugs and are not recommended for production use. Just like Vanilla, they do not support plugins or mods.
```

## 6. Tuinity

> Tuinity is a fork of Paper that is optimized, it supports spigot and bukkit plugins.

Reviewer 2: Please remove Tuinity from this list.

All of Tuinity's projects and patches will be merged into Paper.

View the image below from PaperMC's announcement channel:
![image](https://user-images.githubusercontent.com/72452922/130375679-3e0d6b74-1f34-405e-912e-0d714427f718.png)

## 7. Forge

> Forge is a Minecraft software that supports forge mods.

Reviewer 2: ```suggestion
[Forge](https://files.minecraftforge.net/net/minecraftforge/forge) is a Minecraft software that allows players to use Forge mods.
```

## 8. Cuberite 

> Cuberite is a Minecraft server software that is optimized and supports plugins, it does **not** support spigot and bukkit plugins.

Reviewer 2: ```suggestion
[Cuberite](https://cuberite.org/) is an optimized Minecraft server software used for Cuberite plugins.
```
